src/main/java/org/apache/sling/feature/maven/mojos/ApisJarMojo.java

```diff
@@ -871,6 +880,9 @@ private void onArtifact(final ApisJarContext ctx, Artifact artifact) throws Mojo
                     ApisUtil.buildJavadocClasspath(getLog(), repositorySystem, mavenSession, artifact.getId())
                             .forEach(ctx::addJavadocClasspath);
                 }
+
+            } else {
```

Comment:
If this is important, we could add allow to add metadata like "api-export-capabilities":"true" to a bundle. Otherwise I would just remove the comment